### ISSUE TITLE
Feature Request: "Download" revision code button

### ISSUE THREAD

**Issue by segevd-opti** [Reviewable/Reviewable#1080]
The docs offer an easy way to review revision code locally https://docs.reviewable.io/tips.html#easy-local-revision-checkout.
It would be nice if the UI had a "Download"/"Checkout" button which copies the relevant snippet, e.g
`git fetch origin refs/reviewable/pr123/r2 && git checkout FETCH_HEAD` .

For reference, gerrit has a [similar feature](https://gerrit-review.googlesource.com/Documentation/user-review-ui.html#download).

**Comment by pkaminski** [comment]
Good idea and very easy to implement -- the only question is where to shoehorn it into the UI.  I think we'll have a better space for such widgets as we get into the sidebar redesign, where PR info will get its own full panel.
